clang/test/CIR/CodeGen/atomic-runtime.cpp

```diff
@@ -0,0 +1,309 @@
+// RUN: %clang_cc1 -std=c++17 -triple x86_64-unknown-linux-gnu -fclangir -emit-cir %s -o %t.cir
+// RUN: FileCheck --input-file=%t.cir %s
+
+// Test __atomic_* built-ins that have a memory order parameter with a runtime
+// value.  This requires generating a switch statement, so the amount of
+// generated code is surprisingly large.
+//
+// Only a representative sample of atomic operations are tested: one read-only
+// operation (atomic_load), one write-only operation (atomic_store), one
+// read-write operation (atomic_exchange), and the most complex operation
+// (atomic_compare_exchange).
+
+int runtime_load(int *ptr, int order) {
+  return __atomic_load_n(ptr, order);
+}
```

Comment:
Can you please add a test for the LLVM output just for `runtime_load` (example: clang/test/CIR/CodeGen/abstract-cond.c)? One check should be enough to prove the point, I want to make sure we test this to see how similar it is to OG.

Reply:
Done.